stock_move_auto_assign/models/stock_move.py

```diff
@@ -54,10 +54,12 @@ def _prepare_auto_assign(self, location_field):
         for product_id, location_ids in product_locs.items():
             if not location_ids:
                 continue
-            self._enqueue_auto_assign(
+            job = self._enqueue_auto_assign(
                 self.env["product.product"].browse(product_id),
                 self.env["stock.location"].browse(location_ids),
-            ).delay()
+            )
+            if job:
+                job.delay()
```

Comment:
This is unnecessary. `_enqueue_auto_assign` always will return a job.

Reply:
No there will be no job returned for `_enqueue_auto_assign_auto_release` but for `_enqueue_auto_assign` it will still return the job.

Reply:
I dropped that change